django_api_forms/fields.py

```diff
@@ -30,17 +30,18 @@ def has_changed(self, initial, data):
 
 class FieldList(Field):
     default_error_messages = {
-        'not-list': _('This field have to be a list of objects!'),
+        'not_field': _('Invalid Field type passed into FieldList!'),
+        'not_list': _('This field needs to be a list of objects!'),
     }
 
     def __init__(self, field, **kwargs):
+        super().__init__(**kwargs)
```

Comment:
Had to move this up to the top of `__init__` because Django needs to do some magic to add `default_error_messages` to `self.error_messages`.